motionplan/motionPlanner.go

```diff
@@ -108,6 +108,16 @@ func (req *PlanRequest) validatePlanRequest() error {
 		}
 	}
 
+	_, ok := req.Options["planDeviationMM"].(float64)
```

Comment:
This should be able to go away when this is merged with GoalThreshold as discussed below.